docs/advertising.md

```diff
@@ -0,0 +1,149 @@
+# Ad Layout and Waterfalling
+
+Ad layout is a new config option lets the application to sets a scheme of vasts (instead of VMAP) with waterfalling mechanism.
+
+## Table of Contents
+
+- [Basic Usage](#basic-usage)
+  - [Single Ad](#single-ad)
+  - [Ad Pod](#ad-pod)
+- [Waterfalling](#waterfalling)
```

Comment:
I think we can drop the basic usage section and flatten the TOC to
- Single Ad
- Ad Pod
- Waterfalling